src/quantization/fp32_quantizer_test.cpp

```diff
@@ -36,14 +36,12 @@ TestQuantizerEncodeDecodeMetricFP32(uint64_t dim, int count, float error = 1e-5)
 }
 
 TEST_CASE("encode&decode [ut][fp32_quantizer]") {
-    constexpr MetricType metrics[3] = {
-        MetricType::METRIC_TYPE_L2SQR, MetricType::METRIC_TYPE_COSINE, MetricType::METRIC_TYPE_IP};
-    float error = 1e-5f;
+    constexpr MetricType metrics[3] = {MetricType::METRIC_TYPE_L2SQR, MetricType::METRIC_TYPE_IP};
```

Comment:
for cosine, we only store the normalized vector, because the norm of the vector make no sense for the cosine compute. but in this way, we can't guarantee the consistency after encoding and decoding. So we ignore the consistency for the cosine metric type